libs/labelbox/src/labelbox/schema/project_overview.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, Optional
+from labelbox import pydantic_compat
+
+class Project_Overview(pydantic_compat.BaseModel):
+    """
+    Class that represents a project overview.
```

Comment:
Do you think copilot can generate the detailed params comments here?

Reply:
I have added a few comments to provide additional details.